src/components/CytoscapeGraph/CytoscapeGraphUtils.ts

```diff
@@ -1,4 +1,8 @@
 export const CyEdge = {
+  grpc: 'grpc',
+  grpcErr: 'grpcErr',
+  grpcPercentErr: 'grpcPercentErr',
+  grpcPercentReq: 'grpcPercentReq',
   http: 'http',
   http3XX: 'http3XX',
   http4XX: 'http4XX',
```

Comment:
In all of the Go code on server-side, these do not use capital-X - they are all lowercase "http3xx, http4xx" - but yet here I see http3XX and http4XX and http5XX.

Even if this still works (which I am not convinced just by reading this code), we should be consistent -- use lowercase "x" everywhere, server and client side.